pipelines/snap-atac/bc-attacher.submit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cromwell-tools submit \
+    --url "https://cromwell.caas-prod.broadinstitute.org" \
+    --collection-name "pipeline-surge" \
+    --service-account-key "${HOME}/pipelines_surge_credentials.json" \
```

Comment:
probably shouldn't be committed to the public repo. Better to just sub `{CREDENTIALS_JSON}` instead of relying on a HOME path

Reply:
We'll clean up those scripts after the surge, we generally don't store helper scripts. As a safety precaution I've added the credentials file name to gitignore